config.py

```diff
@@ -788,13 +817,6 @@
     ```
 """
 
-smooth: Optional[int] = 10
-"""
-Number of iterations for the smoothing of the surface data.
-If None, smooth is automatically defined to fill the surface
-with non-zero values. The default is spacing=None.
-"""
```

Comment:
why did you remove this? it's still useful for group morphing I think.

Reply:
Yes, but in fact, we didn't actually make use of this param. I'd rather add it back at some later time with proper support & tests